### ISSUE TITLE
Naming/FileName for hyphenated gems

### ISSUE THREAD

**Issue by stympy** [testdouble/standard#81]
I have a question about the recommended approach when using Standard in a gem that has a hyphen in the name. For example, if I were to build a honeybadger-extras gem, I'd have `lib/honeybadger-extras.rb`, which would trigger Naming/FileName.  So far I've decided to ignore this in `.standard.yml`, like so:

```yaml
ignore:
  - lib/honeybadger-extras.rb:
    - Naming/FileName
```

Is this the recommended approach to silence the problem about the filename not being snake_case?

**Comment by searls** [comment]
Since the rule is about a convention that can't be applied 100% of the time, I think that's enough to nix this rule.

**Comment by searls** [comment]
Fixed in 0.0.34
